src/App.css

```diff
@@ -140,3 +140,72 @@
   line-height: 70px;
   margin-bottom: 10px;
 }
+
+.input {
+  width: 250px;
+  height: 15px;
+  padding: 5px 10px;
+  border-radius: 12px 0px 0px 12px;
+  border: 1px solid #56597a;
+  font-size: 14px;
+  line-height: 1.1;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 400;
+  color: #000;
+}
+.input:focus-visible {
+  border: 1px solid #32333b;
+  outline: none;
+}
+input::placeholder {
+  font-size: 14px;
+  line-height: 1.1;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 400;
+  color: #000;
+}
+.addButton {
+  width: 70px;
+  height: 27px;
+  border-radius: 0px 12px 12px 0px;
+  text-align: center;
+  font-size: 14px;
+  line-height: 1.1;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
It's also not a good idea to copy paste everything, but for our purposes, you could leave these as they are, since we are learning react (not css and preprocessors)